Dockerfile

```diff
@@ -1,24 +1,21 @@
-FROM westonplatter/ruby-nodejs-postgres:2.3.1
+FROM ruby:2.3.1-alpine
 
-RUN apt-get update -qq \
-  && apt-get install -y \
-    build-essential \
-    libpq-dev \
-    ruby-dev \
-    zlib1g-dev \
-    liblzma-dev \
-    libxslt-dev \
-    libxml2-dev \
-    nodejs \
-    locales
+# Install libraries needed for native extensions
+RUN apk add --no-cache build-base libxml2-dev libxslt-dev postgresql-dev nodejs tzdata
 
-RUN mkdir /app
-WORKDIR /app
+RUN mkdir /womenrising
+WORKDIR /womenrising
 
-ADD Gemfile /app/Gemfile
-ADD Gemfile.lock /app/Gemfile.lock
+# Grab the latest version of Bunlder
```

Comment:
Bundler

Reply:
Fixed!

Reply:
It doesn't look like the commit with your fix is here.

Reply:
Ah I amended the commit that added that line instead of adding a whole new commit to fix the typo. Check the list of commits in the PR and the whole "Files changed" diff. a3c149a220cd8bdd19aa0ff77b0cb459d3dd2211 is the sha of the new commit that replaced the old one with the typo.

See https://github.com/womenrising/womenrising/pull/111/commits/a3c149a220cd8bdd19aa0ff77b0cb459d3dd2211#diff-3254677a7917c6c01f55212f86c57fbfR9

Reply:
I see, I was relying on Github collapsing the comment once it changed, and it didn't that time. Thanks!